### DISCUSSION TITLE
Issue with Content-Type during Introspection

### DISCUSSION THREAD

**Discussion by simoneumbro** [wallarm/api-firewall#36] [Q&A]
When the API-Firewall receives an API request and need to validate the access token , it send a POST to the Token Introspection Endpoint and the Content-Type is set to application/octet-stream by default. 
My OAuth2.0 server will accept just application/x-www-form-urlencoded.
Any idea if API-Firewall can set a different Content-Type?

Regards,
Simone

**Comment by afr1ka** [comment]
Hello Simone,

Thank you for your patience. Content-Type of the request to Introspection Endpoint can be changed in  "APIFW_SERVER_OAUTH_INTROSPECTION_CONTENT_TYPE" environment variable. This feature is available right now in the main branch and in the release v0.6.8 which would be a little bit later.

Thanks.
